### DISCUSSION TITLE
Cron Extension for "Last Day of Month"

### DISCUSSION THREAD

**Discussion by eifinger** [jobrunr/jobrunr#294] [Ideas]
Micronaut offers several CRON Extensions described here: https://docs.micronaut.io/3.2.3/api/io/micronaut/scheduling/cron/CronExpression.html

We are currently trying to migrate to jobrunr but we are using the extension "L" to trigger jobs at the last day of the month(28th, 29th, 30th or 31st).

@rdehuyss would you be open for a PR adding this functionality?

**Comment by rdehuyss** [comment]
Yes, off-course.

**Comment by rdehuyss** [comment]
Please base yourself on the PR about durations.

**Comment by rdehuyss** [comment]
@eifinger are you guys working on this? I'm just asking to plan some releases. This could be part of JobRunr 5.

**Reply by eifinger** [reply]
@rdehuyss I am still planning to do it. The last time I gave it a shot I couldn't fully wrap my head around the CRON implementation. Will try again end of next week. If you plan to release please don't wait on me as I cannot promise anything right now 😉

**Comment by rdehuyss** [comment]
This will be part of JobRunr 5.0

**Reply by rkuzmych** [reply]
Sorry, but is there approximate date for v5 release? Thank you in advance

**Reply by rdehuyss** [reply]
Somewhere before Q2 (before end of March).

**Comment by rdehuyss** [comment]
Hi @eifinger , @rkuzmych ,

Jobrunr 5.0.0-RC1 is out with this requested feature. 

Could you guys do a sanity test and provide me with feedback? That way, I can release JobRunr 5.0.0.

Thanks,
Ronald

**Reply by rkuzmych** [reply]
Sorry for the delay.
It was @eifinger who asked about this feature. I just was curious  when would be V5 released :)
